packages/types/src/apps/definition.ts

```diff
@@ -20,6 +20,11 @@ export enum AppNameDefinitions {
   OCULAR_API = "ocular-api",
 }
 
+export enum AppAuthStrategy {
```

Comment:
Rename this AuthStrategy

Reply:
can you do it on your end please , as generating migration gives problem on my laptop and docker takes a lot time 
to build as well

Reply:
I think its best you do it from that end. You've been working on this feature and know way around the code. I'l probably have to end up debugging your code which is a bit inefficient as I am working on other things here.

Reply:
How about you keep it this way in this PR , next PR I will push will have the required migration and renamed enum , because it will take a lot of time if I start generating migration now , will create a small ticket for it as well in asana , what do you think ?

Reply:
Created a Ticket in asana for it !